.gitignore

```
@@ -1,2 +1,2 @@
 .idea
-technical-on-boarding-chart/Chart.yaml
+technical-on-boarding/Chart.yaml
```

why are we ignoring this file?  I think its a good thing to have in the repo?

---

It gets (re)generated as part of CI, and if someone *forgets* to list it as an artifact in CI config, it silently defaults to the one in git history. That's trouble as it could publish an outdated version under some conditions.

---

It gets *generated* on the fly from the `build/Chart.yaml.in` file. This is done in `build/build.sh`.

---

neato.  thanks team!